src/GlobalScreen/Scope/Layout/MetaContent/Media/AbstractMediaWithPath.php

```diff
@@ -27,4 +32,8 @@ protected function isContentDataUri(string $content) : bool
         return (bool) preg_match('/^(data:)([a-z\/]*)((;base64)?)(,?)([A-z0-9=]*)$/', $content);
     }
 
+    protected function hasContentParameters(string $content): bool
```

Comment:
This also violates `PSR-2 + X`

Reply:
This `+ X` was a very expensive decision ;-) I'll fix that

Reply:
I agree :-).